src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -148,7 +149,8 @@ public function build(ContainerBuilder $container): void
             ]);
         }
 
-        $container->addCompilerPass(new AssetsContextPass(), PassConfig::TYPE_BEFORE_OPTIMIZATION);
+        $container->addCompilerPass(new PhpConfigReferenceDumpPass());
+        $container->addCompilerPass(new AssetsContextPass());
```

Comment:
You removed `PassConfig::TYPE_BEFORE_OPTIMIZATION` for `AssertsContextPass`. Is it intentional? That seems unrelated.